shop/rest/serializers.py

```diff
@@ -298,11 +298,22 @@ def get_cart(self, instance):
 
 
 class CustomerSerializer(serializers.ModelSerializer):
-    salutation = serializers.CharField(source='get_salutation_display')
+    number = serializers.CharField(source='get_number')
 
     class Meta:
         model = CustomerModel
-        fields = ('salutation', 'first_name', 'last_name', 'email', 'extra',)
+        exclude = ('user', 'recognized', 'last_access')
+
+    def get_field_names(self, declared_fields, info):
+        fields = set(super(CustomerSerializer, self).get_field_names(declared_fields, info))
```

Comment:
Do we have to call `set()` here? Are we afraid that the parent class may return duplicate field names? Or is this just for the convenience of being able to call `update()`?